docker-compose-postgres.yml

```diff
@@ -8,8 +8,8 @@ services:
     restart: always
     env_file:
       - .envs/db.postgres
-    expose:
-      - 5432
+    ports:
+      - "127.0.0.1:5432:5432"
```

Comment:
If this is the configuration file people use, I'd rather not limit this to local host. Rather I think we should create a docker compose file for tests online only